hudi-spark-datasource/hudi-spark/src/test/java/HoodieJavaApp.java

```diff
@@ -161,7 +161,7 @@ public void run() throws Exception {
         .option(DataSourceWriteOptions.ASYNC_COMPACT_ENABLE().key(), "false")
         .option(DataSourceWriteOptions.ASYNC_CLUSTERING_ENABLE().key(), "true")
         .option(DataSourceWriteOptions.ASYNC_CLUSTERING_ENABLE().key(), "true")
-        .option(DataSourceWriteOptions.ENABLE_OPTIMIZED_SQL_WRITES().key(), "true")
+        .option(DataSourceWriteOptions.SPARK_SQL_OPTIMIZED_WRITES().key(), "true")
```

Comment:
for defaults, lets try to use the actual default instead of hard coding it 
DataSourceWriteOptions.SPARK_SQL_OPTIMIZED_WRITES().default()

Reply:
Fixed.